test/runtests.jl

```diff
@@ -2370,7 +2370,7 @@ else  # run HiGHS tests
             # Boiler serves all of the DHW load, no DHW thermal reduction due to GHP retrofit
             boiler_served_mmbtu = sum(results["ExistingBoiler"]["thermal_production_series_mmbtu_per_hour"])
             expected_boiler_served_mmbtu = 3000 * 0.8 # (fuel_mmbtu * boiler_effic)
-            @test round(boiler_served_mmbtu, digits=1) ≈ expected_boiler_served_mmbtu atol=1.0
+            @test round(boiler_served_mmbtu, digits=1) ≈ expected_boiler_served_mmbtu atol=1.5
```

Comment:
@Bill-Becker  similar to the test above, the only difference I could see between these inputs and previous ones were the previously-incorrect scaling of the last timestep of each month. However, I don't know why the correction results in `boiler_served_mmbtu` coming to 2401.5 rather than the expected 2400